src/inotify.rs

```diff
@@ -131,7 +131,13 @@ impl EventLoop {
         let mut events = mio::Events::with_capacity(16);
         loop {
             // Wait for something to happen.
-            self.poll.poll(&mut events, None).expect("poll failed");
+            match self.poll.poll(&mut events, None) {
+                Err(ref e) if matches!(e.kind(), std::io::ErrorKind::Interrupted) => {
+                    // System call was interrupted, we will retry
```

Comment:
I'd like to land this but not loose the information, could you just document in the comment that there is no actual test for this ?